static/results.js

```diff
@@ -13,87 +21,87 @@ $(document).ready(function(){
       }
       if ($(this).html().includes('User Agent')) {
         var filename = '/static/results-text/user-agent.txt';
-        var selfname = $(this);
+        var selfname = $(this).prepend('<span>1header <span>');
```

Comment:
How is this used?  It seems like an antipattern to sort via the HTML inside a span.  Maybe we can use a data attribute, like `<span data-sort-group='1'>`

Reply:
good suggestion, will do.